web/src/search_suggestion.ts

```diff
@@ -768,7 +802,24 @@ class Attacher {
 
     attach_many(suggestions: Suggestion[]): void {
         for (const suggestion of suggestions) {
-            const suggestion_line = [...this.base, suggestion];
+            let suggestion_line;
+            if (this.base.length === 0) {
+                suggestion_line = [suggestion];
+            } else {
+                // When we add a user to a user group, we
+                // replace the last pill.
```

Comment:
This feels a bit janky, but I think it would take a significant refactor to have more structured data here and I've been trying to keep the changes simpler where I can.